src/lsp/main.ts

```diff
@@ -411,40 +410,61 @@ function registerEventHandlers(
     );
 }
 
-async function startClient(
-    clojureLspPath: string,
-    context: vscode.ExtensionContext,
-    handler: TestTreeHandler
-): Promise<void> {
+function stopClient() {
+    const client = getStateValue(LSP_CLIENT_KEY);
+    void vscode.commands.executeCommand(
+        'setContext',
+        'clojureLsp:active',
+        false
+    );
+    updateStatusItem('stopped');
+    setStateValue(LSP_CLIENT_KEY, undefined);
+    if (client) {
+        return client.stop();
+    }
+}
+
+async function startClientCommand() {
+    await maybeDownloadLspServer();
+    lspStatus.show();
+    void startClient();
```

Comment:
Is there a reason to not await this here?

Reply:
Depends on where is ”this” 😄 If you mean the download, then yes. But that's so obvious, so I'm going to assume `startClient()`. Dunno. Was thinking this is always called in fire-and-forget manner, but maybe it can be used in a promise chain? If so we should await. I'll fix that.